specs/unified_kernel_image.md

```diff
@@ -31,9 +31,11 @@ The file format is PE/COFF (Portable Executable / Common Object File Format). Th
 industry-standard file format, used for example in UEFI environments,
 and UKIs follow the standard, so exact details will not be repeated here.
 
-UKIs can be generated via a [single, relatively simple `objcopy`
-invocation](https://www.freedesktop.org/software/systemd/man/systemd-stub.html#Assembling%20Kernel%20Images),
-that glues the listed components together, generating one PE binary that then can be signed for SecureBoot.
+UKIs are created as a PE/COFF file with additional sections that glues together all the components.
```

Comment:
"with additional sections that glues together all the components." => I don't really get what this is supposed to say, maybe try to reword?

Reply:
Reworded. It's not so easy to keep this short but not introduce inaccuracies.